projects/com.oracle.truffle.llvm.test/tests/c/atexit/atexit003.c

```diff
@@ -0,0 +1,9 @@
+#include <stdlib.h>
+#include <stdio.h>
+
+static void exit1() { printf("something went wrong!\n"); }
```

Comment:
Could you change this to be more descriptive like 'error: functions registered using atexit should only be executed after successful program execution' or something of the like?